apps/test/unit/templates/sectionProgressV2/ProgressFeedbackBannerTest.jsx

```diff
@@ -0,0 +1,221 @@
+      i18n.progressV2_feedback_question()
+    );
+    const shareMoreText = screen.queryByText(
+      i18n.progressV2_feedback_shareMore()
+    );
+    expect(questionText).to.not.exist;
+    expect(shareMoreText).to.not.exist;
+  });
+
+  it('clicking thumbs up attempts to send feedback and asks for more detailed feedback', async () => {
+    render(
+      <UnconnectedProgressFeedbackBanner
+        {...defaultProps}
+        createProgressV2Feedback={fakeCreate}
+      />
+    );
+    const thumbsUpButton = screen.getByTitle(
+      i18n.progressV2_feedback_thumbsUp()
+    );
+    expect(thumbsUpButton).to.be.visible;
+    fireEvent.click(thumbsUpButton);
+
+    await waitFor(() => {
+      expect(fakeCreate).to.have.been.calledOnce;
+    });
+    const shareMoreText = screen.getByText(
+      i18n.progressV2_feedback_shareMore()
+    );
+    const shareMoreLink = screen.getByRole('link', {
+      name: i18n.progressV2_feedback_shareMoreLinkText(),
+    });
+    expect(shareMoreLink).to.be.visible;
+    expect(shareMoreText).to.be.visible;
+  });
+
+  it('clicking thumbs down attempts to send feedback and asks for more detailed feedback', async () => {
+    render(
+      <UnconnectedProgressFeedbackBanner
+        {...defaultProps}
+        createProgressV2Feedback={fakeCreate}
+      />
+    );
+    const thumbsDownButton = screen.getByTitle(
+      i18n.progressV2_feedback_thumbsDown()
+    );
+    expect(thumbsDownButton).to.be.visible;
+    fireEvent.click(thumbsDownButton);
+    await waitFor(() => {
+      expect(fakeCreate).to.have.been.calledOnce;
+    });
+
+    const shareMoreText = screen.getByText(
+      i18n.progressV2_feedback_shareMore()
+    );
+    const shareMoreLink = screen.getByRole('link', {
+      name: i18n.progressV2_feedback_shareMoreLinkText(),
+    });
+    expect(shareMoreLink).to.be.visible;
+    expect(shareMoreText).to.be.visible;
+  });
+
+  it('user is able to close the banner', async () => {
+    render(
+      <UnconnectedProgressFeedbackBanner
+        {...defaultProps}
+        createProgressV2Feedback={fakeCreate}
+      />
+    );
+
+    // Give feedback
+    const thumbsUpButton = screen.getByTitle(
+      i18n.progressV2_feedback_thumbsUp()
+    );
+    expect(thumbsUpButton).to.be.visible;
+    fireEvent.click(thumbsUpButton);
+    await waitFor(() => {
+      expect(fakeCreate).to.have.been.calledOnce;
+    });
+
+    // Close the banner
+    const closeButton = screen.getByText('×');
+    expect(closeButton).to.be.visible;
+    fireEvent.click(closeButton);
+    await waitFor(() => {
+      const questionText = screen.queryByText(
+        i18n.progressV2_feedback_question()
+      );
+      const shareMoreText = screen.queryByText(
+        i18n.progressV2_feedback_shareMore()
+      );
+      expect(questionText).to.not.exist;
+      expect(shareMoreText).to.not.exist;
+    });
+  });
+
+  it('attempts reset and reload if error from createFeedback', async () => {
```

Comment:
@lfryemason I had a hard time with this test.  After some research, I am thinking that perhaps how this is implemented in the component might benefit from some refactoring.